system-test/read-rows.ts

```diff
@@ -123,97 +168,95 @@ describe('Bigtable/Table', () => {
     });
   });
 
-  describe('createReadStream', () => {
-    let clock: sinon.SinonFakeTimers;
-    let endCalled: boolean;
-    let error: ServiceError | null;
-    let requestedOptions: Array<{}>;
-    let responses: Array<{}> | null;
-    let rowKeysRead: Array<Array<{}>>;
-    let stub: sinon.SinonStub;
-
-    beforeEach(() => {
-      clock = sinon.useFakeTimers({
-        toFake: [
-          'setTimeout',
-          'clearTimeout',
-          'setImmediate',
-          'clearImmediate',
-          'setInterval',
-          'clearInterval',
-          'Date',
-          'nextTick',
-        ],
+  describe('createReadStream using mock server', () => {
```

Comment:
![image](https://github.com/googleapis/nodejs-bigtable/assets/8935272/02e2372f-27bb-4592-ac96-af159bc8f4d4) It must be any or unknown according to the linter and it can't be unknown because it has to have the message property. Any specific suggestions?

Reply:
if you remove the "any" you can then add an assertion that it's of type Error (or GoogleError if it comes from gax?)

```
catch(err){
    assert(err instanceof Error);
    assert.strictEqual(err.message, 'The client has already been closed.');
}
```

Reply:
Done